src/run-tests-in-page.js

```diff
@@ -0,0 +1,28 @@
+const puppeteer = require("puppeteer");
+const runTestcases = require("./run-testcases");
+const { concatReport } = require("./axe-reporter-earl");
+
+async function runTestsInPage(program, toolRunner) {
+  const { page, browser } = await puppeteerSetup();
+  const testResults = await runTestcases(program, testcase => {
+    return toolRunner(page, testcase);
+  });
+
+  await stopPuppeteer({ page, browser });
+  return concatReport(testResults);
```

Comment:
```suggestion
  const puppeteerInstance = await puppeteerSetup();
  const testResults = await runTestcases(program, testcase => {
    return toolRunner(puppeteerInstance.page, testcase);
  });

  await stopPuppeteer(puppeteerInstance);
  return concatReport(testResults);
```

Reply:
There is no such thing as a "puppeteerInstance". That's why I didn't give it a name, because that collection of two objects isn't anything other than a return value.